packages/suite-data/browser-detection.webpack.ts

```diff
@@ -8,6 +8,9 @@ module.exports = {
         path: path.resolve(__dirname, 'files/browser-detection'),
         filename: 'index.js',
     },
+    resolve: {
+        extensions: ['.ts', '.tsx', '.js', '.jsx'],
```

Comment:
nit: tsx/jsx in not really needed here (no react here)

Reply:
https://github.com/trezor/trezor-suite/pull/5487/commits/d32855c63de484f7f02185dad165a195ff0fc6ce